apps/challenges/challenge_config_utils.py

```diff
@@ -242,15 +242,7 @@ def get_value_from_field(data, base_location, field_name):
     "leaderboard_deletion_after_creation": "ERROR: The leaderboard with ID: {} not found in config. Deletion of an existing leaderboard after challenge creation is not allowed.",
     "missing_leaderboard_labels": "ERROR: There is no 'labels' key in the schema for the leaderboard with ID: {}.",
     "missing_challenge_phases": "ERROR: No challenge phase key found. Please add challenge phases in the YAML file and try again!",
-    "missing_challenge_phase_codename": "ERROR: No codename found for the challenge phase. Please add the codename and try again!",
-    "missing_test_annotation_file": "ERROR: No test annotation file found in the zip file for challenge phase {}.",
-    "missing_submission_meta_attribute_keys": "ERROR: Please enter the following keys to the submission meta attribute in challenge phase {}: {}",
```

Comment:
Why are we removing these? These need to be tested. Have you removed them in your PR? If yes, please find appropriate places to add these tests back.